src/shared/components/ui/Calendar/Calendar.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import { isWeekend, today } from "@internationalized/date";
+import type { Meta, StoryObj } from "@storybook/react";
+import { I18nProvider, useLocale } from "react-aria-components";
+
+import { currentTimezone } from "~/utils";
+import { Calendar } from ".";
+
+const meta: Meta<typeof Calendar> = {
+  title: "Components/Datetime/Calendar",
+  component: Calendar,
+  decorators: [
+    (Story) => (
+      <I18nProvider locale='en-US'>
+        <Story />
+      </I18nProvider>
+    ),
+  ],
+  parameters: {
+    componentSubtitle:
+      "A calendar displays a date grid and allows users to select a single date.",
+  },
+  args: {},
+  argTypes: {},
+  tags: ["autodocs"],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Calendar>;
+
+export const Default: Story = {};
+
+export const DifferentLocale: Story = {
+  decorators: [
+    (Story) => (
+      <I18nProvider locale='es-UY'>
+        <Story />
+      </I18nProvider>
+    ),
+  ],
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "To change the language on the calendar labels, just change the locale on the `I18nProvider`.",
+      },
+    },
+  },
+  render: () => <Calendar />,
+};
+
+export const DisabledWeekends: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Disabling dates is achieved by passing a filtering function to the `isDateUnavailable` prop. It receives each `dateValue`, and you can run that through any check. The example uses `isWeekend`, but the same principle applies to disabling dates before today or other use cases. You can use any of the checking functions from `@internationalized/date` or compare dates using `dateValue.compare()`",
+      },
+    },
+  },
+  render: () => {
+    const { locale } = useLocale();
+
+    return (
+      <Calendar
+        isDateUnavailable={(dateValue) => isWeekend(dateValue, locale)}
+      />
+    );
+  },
+};
+
+export const DisabledBeforeToday: Story = {
+  render: () => {
+    return (
+      <Calendar
+        isDateUnavailable={(dateValue) =>
+          dateValue.compare(today(currentTimezone)) < 0
+        }
+      />
+    );
+  },
+};
```

Comment:
I don't think we should add these stories, they are actually use cases of your component but you are showing that isDataUnavailable works 🤔

Reply:
Im talking aabout the date picker as a whole component. At the end, thats what stories are for